### PR TITLE
Fix https-invalid landing page URL link in README

### PR DESCRIPTION
**Pull request by neonsoftware**
#### Problem
Users currently face a Browser Security warning blockage (invalid https), when opening the project's landing page link on the README.

#### Summary of Changes

Using directly the new URL https://buildwithmatter.com/ instead of the current problematic URL redirection.

### CODE REVIEW

**Review by Damian-Nordic — APPROVED** [review]

**Comment by neonsoftware** [comment]
Am I supposed to do any modifications related to the "restiled" associated PR, or it will be ok ?

**Comment by Damian-Nordic** [comment]
@neonsoftware yes, you need to apply changes made by restyler to your branch before it can be merged. You can do it manually if the change is small, or merge the PR created by restyler to your branch.

**Comment by neonsoftware** [comment]
@Damian-Nordic - I have manually applied the change done by restyler to this commit (new line), do I now need a maintaner to re-trigger the actions ?  (look like from https://github.com/project-chip/connectedhomeip/actions/runs/849446105  "This workflow is awaiting approval from a maintainer in #6860")

**Comment by neonsoftware** [comment]
@Damian-Nordic  - I respond to my-self: restyled seems to automatically self-retrigger, just with some delay. Will wait.

**Review by andy31415 — APPROVED** [review]

**Review by jelderton — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

**Comment by neonsoftware** [comment]
Is there anything else that I cloud do for this patch ?

**Pull request merged**
